src/libraries/System.Threading.RateLimiting/src/System/Threading/RateLimiting/ConcurrencyLimiter.cs

```diff
@@ -39,6 +39,14 @@ public sealed class ConcurrencyLimiter : RateLimiter
         public ConcurrencyLimiter(ConcurrencyLimiterOptions options)
         {
             _options = options ?? throw new ArgumentNullException(nameof(options));
+            if (options.PermitLimit <= 0)
+            {
+                throw new ArgumentException($"{nameof(options.PermitLimit)} must be set to values greater than 0.");
+            }
+            if (options.QueueLimit < 0)
+            {
+                throw new ArgumentException($"{nameof(options.QueueLimit)} must be set to values greater than or equal to 0.");
```

Comment:
```suggestion
                throw new ArgumentException($"{nameof(options.QueueLimit)} must be set to a value greater than or equal to 0.");
```

Reply:
Same for other limiters, they're inconsistent right now.

Reply:
Just saw this as well, will fix up after CI finishes